web/src/components/JSMpegPlayer.jsx

```diff
@@ -24,7 +40,22 @@ export default function JSMpegPlayer({ camera, width, height }) {
       }
     }
 
+    const fullscreenchange = () => {
+      if (document.webkitIsFullScreen) {
+        return;
+      }
+
+      if (wakeLock === null) {
+        return;
+      }
+
+      wakeLock.release().then(() => {
+        wakeLock = null;
+      });
+    }
+
     video.els.canvas.addEventListener('click',fullscreen)
+    video.els.canvas.addEventListener('webkitfullscreenchange',fullscreenchange)
```

Comment:
also, you might want to add `removeEventListener` such as 
```
    return () => {
      video.els.canvas.removeEventListener('click',fullscreen)
      video.els.canvas.removeEventListener('webkitfullscreenchange',fullscreenchange)
      video.destroy();
    };
```

Reply:
Done. Good suggestion.